database/knexfile.js

```diff
@@ -9,7 +9,14 @@ module.exports = {
     schemaName: env.PGSCHEMA,
   },
   pool: {
-    min: process.env.KNEX_POOL_MIN || 1,
-    max: process.env.KNEX_POOL_MAX || 100,
+    min: +process.env.KNEX_POOL_MIN || 0,
+    max: +process.env.KNEX_POOL_MAX || 5,
+    
+    // tarn config (https://www.npmjs.com/package/tarn)
+    propagateCreateError: false,
```

Comment:
@k1ch do you want to "hard-code" these values or make them available to be set as environment variables with these default values if not specified?

Reply:
I can introduce the ENVs but I doubt we will use them.